src/wizard/javascript/browser.md

```diff
@@ -5,28 +5,39 @@ support_level: production
 type: language
 ---
 
-The quickest way to get started is to use the CDN hosted version of the JavaScript browser SDK:
-
-```html
-<script
-  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/bundle.min.js"
-  integrity="sha384-{{ packages.checksum('sentry.javascript.browser', 'bundle.min.js', 'sha384-base64') }}"
-  crossorigin="anonymous"
-></script>
+Install our JavaScript browser SDK using either `yarn` or `npm`:
+
+```bash {tabTitle: ESM}
+# Using yarn
+yarn add @sentry/browser @sentry/tracing
+# Using npm
+npm install --save @sentry/browser @sentry/tracing
 ```
 
-You should `init` the Sentry Browser SDK as soon as possible during your page load:
+We also support alternate [installation methods](/platforms/javascript/install/).
+
+`init` the Sentry Browser SDK as soon as possible during your page load:
 
 ```javascript
-Sentry.init({ dsn: "___PUBLIC_DSN___" });
+import * as Sentry from "@sentry/browser";
+import { Integrations } from "@sentry/tracing";
+
+Sentry.init({
+  dsn: '___PUBLIC_DSN___',
+  integrations: [
+    new Integrations.BrowserTracing(),
+  ],
+
+  // We recommend adjusting this value in production, or using tracesSampler
```

Comment:
> or using tracesSampler

This seems tricky to introduce at this stage. What is `tracesSampler`? How do I use it? Where do I find out more about it?

I'd sooner put a note that says (perhaps outside of the code snippet) that says: "It is recommended to lower this value in production. You can find out more about configuring sampling in our full documentation."

Reply:
In our docs, we want everything in the code snippet. But here, I think it makes sense to pull it out. As you note, too much isn't useful in the absence of supporting content.

Will edit to:
"We recommend lowering this value in production. Learn more about configuring sampling in our full documentation." with a link to https://docs.sentry.io/platforms/javascript/performance/sampling/